### PR TITLE
Provide more information in the notify email

### PR DESCRIPTION
**Pull request by hv0905**
Currently the notify email just provide the count of unread messages. This is inconvenient when I want to know who is calling me.

### CODE REVIEW

File: Kahla.sln.DotSettings [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (Anduin2017) [inline Kahla.sln.DotSettings]: Do NOT check in this.

File: Kahla.Server/Services/EmailNotifier.cs [hunk 2]
```diff
@@ -90,27 +89,39 @@ public async Task<string> BuildEmail(KahlaUser user, KahlaDbContext dbContext, I
...
                 var currentUnread = conversation.GetUnReadAmount(user.Id);
-                if (currentUnread > 0)
-                {
-                    totalUnread += currentUnread;
-                    inConversations++;
+
+                if (currentUnread <= 0) continue;
+                totalUnread += currentUnread;
+                inConversations++;
+                if (inConversations >= 50) {
```
Comment (Anduin2017) [inline Kahla.Server/Services/EmailNotifier.cs]: These two `if` can be simplified.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]

File: Kahla.sln.DotSettings (hunk 1 above)
Comment (EdiWang) [reply Kahla.sln.DotSettings]: Ignore this file in gitignore

**Review by EdiWang — CHANGES_REQUESTED** [review]
Add Kahla.sln.DotSettings to .gitignore

**Comment by hv0905** [comment]
> Add Kahla.sln.DotSettings to .gitignore

.DotSettings is a team-shared file and shouldn't be added into `.gitignore`.. I will just deleted it,

**Review by Anduin2017 — APPROVED** [review]

**Pull request merged**